solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Features/Details/Scenes/LaunchDetails/Views/LaunchDetailsBodyView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  LaunchDetailsBodyView.swift
+//  SpaceApp
+//
+//  Created by Razee Hussein-Jamal on 08/03/2022.
+//
+
+import SwiftUI
+
+struct LaunchDetailsBodyView: View {
+    let title: String
+    let description: String
+    let buttonTitle: String
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 10) {
+            Text(title)
+                .font(.title2)
+                .bold()
+                .frame(alignment: .leading)
+            
+            Text(description)
+                .lineLimit(3)
```

Comment:
**[SUGGESTION]**
Consider also storing this magic number into a contextual enum like:
```swift
enum Constants {
    static let descriptionLineLimit: Int = 3
}
```

Then:
```suggestion
                .lineLimit(Constants.descriptionLineLimit)
```